tests/function/gtOpenClose.cpp

```diff
@@ -0,0 +1,269 @@
+/*++
+
+INTEL CONFIDENTIAL
+Copyright 2016 - 2017 Intel Corporation
+
+The source code contained or described  herein and all documents related to
+the  source  code  ("Material")  are  owned  by  Intel  Corporation  or its
+suppliers  or  licensors.  Title   to  the  Material   remains  with  Intel
+Corporation or  its suppliers  and licensors.  The Material  contains trade
+secrets  and  proprietary  and  confidential  information  of Intel  or its
+suppliers and licensors.  The Material is protected  by worldwide copyright
+and trade secret laws and treaty provisions. No part of the Material may be
+used,   copied,   reproduced,   modified,   published,   uploaded,  posted,
+transmitted,  distributed, or  disclosed in  any way  without Intel's prior
+express written permission.
+
+No license under any patent, copyright,  trade secret or other intellectual
+property  right  is  granted to  or conferred  upon  you by  disclosure  or
+delivery of the  Materials, either  expressly, by  implication, inducement,
+estoppel or otherwise. Any license  under such intellectual property rights
+must be express and approved by Intel in writing.
+
+--*/
+
+#include <opae/access.h>
+
+#include "common_test.h"
+#include "gtest/gtest.h"
+#include "types_int.h"
+
+using namespace common_test;
+using namespace std;
+
+class LibopaecOpenFCommonHW : public common_test::BaseFixture,
+                              public ::testing::Test {};
+
+class LibopaecCloseFCommonHW : public common_test::BaseFixture,
+                               public ::testing::Test {};
+
+TEST_F(LibopaecOpenFCommonHW, 03) {
```

Comment:
All tests that have integer only as test name need to be verified with mock driver first.